src/test/Security/EncryptionTest.java

```diff
@@ -64,5 +71,10 @@ public void encryptDecryptFileTest() throws IOException, GeneralSecurityExceptio
     // Delete created file
     returnFile.delete();
   }
-   */
```

Comment:
can we add more tests? I would just be more confident if there were like 10 encryption tests in general. Maybe see if we have support for pdfs in terms of encrypting/decrypting them, but also check if our filestream code also allows for encrypting/decrypting images (pngs, jpgs, etc.) and also maybe other file formats? like docx?

Reply:
Will do :)

Reply:
Added a couple more tests